src/js/components/PagedNavComponent.jsx

```diff
@@ -103,7 +103,9 @@ var PagedNavComponent = React.createClass({
     var leftArrow =
       this.props.useArrows ?
         <li className={leftArrowsClassSet}>
-          <button onClick={this.handlePageChange.bind(this, currentPage - 1)}>
+          <button
+            className="btn btn-default btn-sm"
```

Comment:
Let's keep the first property on the same line, if it fits. Same for the `>` button below